core/logger/logger.go

```diff
@@ -88,7 +88,11 @@ func CreateProductionLogger(
 	if err != nil {
 		log.Fatal(err)
 	}
+	sugaredLogger := zl.Sugar()
+	if !devMode {
+		sugaredLogger = sugaredLogger.With("CLInstanceID", static.InstanceUUID, "CLVersion", static.Version, "CLSHA", static.Sha)
```

Comment:
This will add these fields to every line of logging, right? Seems noisy. Could we just log `CLVersion` and `CLSHA` once initially with the instance ID, and then we can always trace back to the first appearance of the instance ID if needed.

Reply:
Agreed.